dashboard/src/pages/apps/new.tsx

```diff
@@ -0,0 +1,505 @@
+    fontSize: '14px',
+    marginLeft: '4px',
+
+    width: '300px',
+
+    display: 'flex',
+    flexDirection: 'column',
+
+    '::placeholder': {
+      color: vars.text.black3,
+    },
+  },
+})
+
+const InputFormCheckBox = styled('div', {
+  base: {
+    background: vars.bg.white1,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    marginLeft: '4px',
+    padding: '8px 12px',
+
+    width: '160px',
+  },
+})
+
+const InputFormWebsite = styled('div', {
+  base: {
+    background: vars.bg.white2,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    marginLeft: '4px',
+    padding: '8px 12px',
+
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '12px',
+  },
+})
+
+const InputFormWebsiteButton = styled('div', {
+  base: {
+    display: 'flex',
+    gap: '8px',
+    marginBottom: '4px',
+    marginLeft: '4px',
+  },
+})
+
+const InputFormRadio = styled('div', {
+  base: {
+    background: vars.bg.white2,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    marginLeft: '4px',
+    padding: '8px 12px',
+
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '12px',
+  },
+})
+
+styled('div', {
+  base: {
+    display: 'flex',
+    borderRadius: '4px',
+    backgroundColor: vars.bg.black1,
+  },
+})
+
+styled('div', {
+  base: {
+    margin: 'auto',
+    color: vars.text.white1,
+    fontSize: '16px',
+    fontWeight: 'bold',
+  },
+})
+
+const RepositoriesContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+  },
+})
+
+interface WebsiteProps {
+  selected: Accessor<string>
+  setSelected: Setter<string>
+}
+
+const Website = (props: WebsiteProps) => {
+  return (
+    <InputFormWebsite>
+      <InputForm>
+        <InputFormText>fqdn</InputFormText>
+        <InputBar placeholder='fqdn' />
+      </InputForm>
+      <InputForm>
+        <InputFormText>Path prefix</InputFormText>
+        <InputBar placeholder='path_prefix' />
+      </InputForm>
+      <InputForm>
+        <InputFormCheckBox>
+          <Checkbox>strip_prefix</Checkbox>
+          <Checkbox>https</Checkbox>
+          <Checkbox>h2c</Checkbox>
+        </InputFormCheckBox>
+      </InputForm>
+      <InputForm>
+        <InputFormText>Http port</InputFormText>
+        <InputBar placeholder='http_port' />
+      </InputForm>
+      <InputForm>
+        <Radio items={authenticationTypeItems} selected={props.selected} setSelected={props.setSelected} />
+      </InputForm>
+    </InputFormWebsite>
+  )
+}
+
+interface WebsitesProps {
+  websites: WebsiteStruct[]
+}
+
+const Websites = (props: WebsitesProps) => {
+  return (
+    <For each={props.websites}>
+      {(website) => {
+        return <Website selected={website.signal[0]} setSelected={website.signal[1]} />
+      }}
+    </For>
+  )
+}
+
+export type WebsiteStruct = {
+  signal: Signal<string>
+  // authenticationType: Accessor<string>
+  // setAuthenticationType: Setter<string>
+}
```

Comment:
WebsiteStructという型の中にSignalがある変な構造になってますが、あまり良くなさそう

この型ではWebsiteの設定に関する情報を保持したいと推測するのですが、そうであれば普通に↓のようにデータ型を書いて、
```suggestion
interface Website {
  fqdn: string
  authenticationType: string
  // その他のフィールド ...
}
```
親（この下の `export default () => {}` 関数の中）で `const [websites, setWebsites] = createSignal<Website[]>([])` して、
Websiteを設定するコンポーネントではwebsiteとsetWebsiteだけ取って、各フィールドの更新時にsetWebsiteするのが良いと思います
各WebsiteコンポーネントにわたすsetWebsiteでは例えば以下のようにそのindexだけ更新するような関数を渡す？
ちょっと僕もよくわかってないから動くかわからない
```
<For each={websites()}>
  {(website, i) => (
    <Website
      website={website}
      setWebsite={(website) => setWebsites((current) => {
        current[i] = website
        return current
      })}
    />
  )}
</For>
```